pkg/arvo/sys/lull.hoon

```diff
@@ -2472,6 +2478,92 @@
       [%$ %whiz ~]
       [@tas %meta vase]
   ==
++$  note-gall                                           ::  userspace note
```

Comment:
`note-gall` is the new type for userspace notes. I'm very interested in thoughts on this list, since I'm definitely not familiar with all the notes included here. I constructed this list by starting with the work @Fang- has been doing on userspace permissions, then adding notes until the `%base` desk compiled. Many of these are only used in `/app/helm` and `/app/herm`.

Note that I tried to make this list with bucgar, but always ran into `mint-vain`, even when including a bucsig.

Reply:
You'll need to do the same for $gift:agent:gall, so that the agent type won't depend (contravariantly) on the full Arvo gift type.

I'm not sure why you'd get a mint-vain for this, but let's figure that out, because it should work.  One guess as to why it's not working would be that there's a mutual recursion in the types somewhere, where they're defined in terms
of each other, and maybe that's causing a problem -- I wouldn't expect a mint-vain from that, though.

Reply:
Figured out the bucgar problem and pushed a fix, I had missed some changes to the types of various tasks that I pulled in when merging to `next/arvo`. Doing the gifts next.

Reply:
👍

Reply:
Gifts are now done.